src/universal/modules/email/containers/TeamInvite/TeamInvite.js

```diff
@@ -33,13 +34,45 @@ const videoGraphicStyle = {
 const videoGraphicSrc = `${emailDir}retro-video-still.png`
 
 const TeamInvite = (props) => {
-  const {inviterName, inviterEmail, inviteeEmail, inviteeName, teamName, inviteLink} = props
+  const {
+    inviterName,
+    inviterEmail,
+    inviteeEmail,
+    inviteeName,
+    meeting,
+    teamName,
+    inviteLink
+  } = props
   const inviteeEmailBlock = (
     <a href={`mailto:${inviteeEmail}`} style={emailCopyStyle}>
       {inviteeEmail}
     </a>
   )
   const nameOrEmail = inviteeName || inviteeEmailBlock
+  const buttonLabel = meeting
+    ? `Join ${meetingTypeToLabel[meeting.meetingType]} Meeting`
+    : 'Join Team'
+  const meetingCopyLabelLookup = {
+    action: 'an Action Meeting',
+    retrospective: 'a Retrospective Meeting'
+  }
+  const activeMeetingCopy = meeting ? (
+    <span>
+      {`) has started ${meetingCopyLabelLookup[meeting.meetingType]} for your team (`}
+      <b>{teamName}</b>
+      {'). Just a few clicks and you’re in!'}
+    </span>
+  ) : (
+    undefined
+  )
+  const defaultCopy = (
+    <span>
+      {') has invited you to join a team ('}
+      <b>{teamName}</b>
+      {') on Parabol.'}
+    </span>
+  )
+  const copy = meeting ? activeMeetingCopy : defaultCopy
```

Comment:
there's 2 conditionals that check for meeting. it'd be easier to just say {meeting ? <ActiveMeeting/> : <DefaultCopy/>}